kqueen/engines/openstack_kubespray.py

```diff
@@ -592,8 +607,11 @@ def _run_ansible(self, inventory="hosts.json", playbook="cluster.yml"):
                 env=env,
             )
             pipe.wait()
-            if pipe.returncode:
-                logger.warning("Non zero exit status from ansible (%s)" % pipe.returncode)
+        if pipe.returncode:
+            with open(self.ansible_log, "r") as log_file:
+                result = log_file.read()
+            if result.find("fatal:"):
```

Comment:
imo its bad case to find `fatal`. it looks like hardcode, can we trigger it on any other points? maybe exit code or smth similar?

Reply:
no, it is ansible expected behavior. we check error code first and then fatal, there are no any names from user input, so it is ok to look for fatal with :

Reply:
okok, agree